base64/base64.go

```diff
@@ -1,28 +1,73 @@
 /*
+	Go base64 - prints the current working directory.
+	Copyright (C) 2015 Robert Deusser
 	This program is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License version 3 as
-	published by the Free Software Foundation.
-
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
```

Comment:
Didn't think about that. For safety, I'll keep the files you merged from his repo with the original license.